README.md

```diff
@@ -0,0 +1,69 @@
+
+## Continuous integration testing
+
+Google Cloud Build supports continuous integration testing, with a test running automatically every time a Pull Request is created or updated in GitHub.
+
+To try this out for yourself with a simple Go application which prints a greeting:
+
+* Fork this repo on GitHub by clicking the "Fork" icon in the top right-hand corner of the screen.
+* Open the [Google Cloud Build GitHub app](https://github.com/apps/google-cloud-build), click Install, and follow the setup instructions.  Select the `hello-kubernetes` repository you just created.
+* Back in your forked repo in GitHub, click to open `main_test.go`.  You will see some Go tests which make sure that the greeting matches the name provided.
+* Click the pencil icon in the top right-hand corner of the screen.
+* Edit one of the greetings so that the greeting does not match the name.
+* At the bottom of the screen, choose "Create a new branch and start a pull request".  Click "Propose file change".
+* On the page which opens, click "Create pull request".
+* You should see a message: "All checks have failed".  If you wish, you can see the build log by clicking the "Details" button and then selecting the link at the bottom of the screen which says "View more details on Google Cloud Build".
+* Next, choose the "Files changed" tab, and click the pencil icon again.
+* Make another edit so that the greetings now match the names.
+* Click the "Commit changes" button at the bottom of the screen.
+* Go back to the "Conversation" tab.
+* You should see that tests pass, and a green check is now returned.
+
+## Automated deployment
+
+Google Cloud Build can be used to deploy to Kubernetes automatically whenever you merge to a particular Git branch or tag.  Here, we're going to deploy to Google Kubernetes Engine (GKE) whenever a merge to `master` occurs.
+
+Before we start, we need to enable the GKE API, create a Kubernetes cluster for testing, and grant deployment permission to the Cloud Build service account.  To do this, copy and paste the following lines into your terminal:
+
+```sh
+gcloud services enable container.googleapis.com
+gcloud container clusters create my-cluster --zone=us-central1-f
+export PROJECT=$(gcloud info --format='value(config.project)')
+export PROJECT_NUMBER=$(gcloud projects describe $PROJECT --format 'value(projectNumber)')
+export CB_SA_EMAIL=$PROJECT_NUMBER@cloudbuild.gserviceaccount.com
+gcloud projects add-iam-policy-binding $PROJECT --member=serviceAccount:$CB_SA_EMAIL --role='roles/container.developer'  
+```
+
+Then:
+
+* Visit the [Build Triggers page](https://console.cloud.google.com/cloud-build/triggers) in the Cloud Console.
+* Click "Create Trigger".
```

Comment:
It's actually "Add Trigger"

Reply:
Looks like it actually changes depending on whether you have existing triggers or not!  Updated.

![add-trigger](https://user-images.githubusercontent.com/9892065/43725006-60987f18-9969-11e8-90b1-9c05f39492ca.png)

![build-triggers](https://user-images.githubusercontent.com/9892065/43724937-3c6ee37a-9969-11e8-92dd-870ae7d2c478.png)